pages/docs/ecosystem-roles/validator/systemd/validator-node.mdx

```diff
@@ -128,42 +123,43 @@ ExecStart=/usr/bin/tangle-standalone \
   --name <NODE-NAME> \
   --chain tangle-testnet \
   --node-key-file "/home/<USERNAME>/node-key" \
-  --port 9944 \
+  --port 30333 \
```

Comment:
```suggestion
  --port 9944 \
```

Lets keep this 9944, its the default for explorer, so helps to quickly verify

Reply:
it will give an error, we can't do `author_rotateKeys` steps if use 9944 there

Reply:
`connection reset by peer` that's the error

Reply:
ah thats weird, lets keep it 30333 then